packages/hoppscotch-cli/src/types/commands.ts

```diff
@@ -1,6 +1,6 @@
 export type TestCmdOptions = {
-  env: string;
-  delay: number;
+  env: string | undefined;
+  delay: string | undefined;
 };
 
 export type HoppEnvFileExt = "json";
```

Comment:
Snake case at https://github.com/hoppscotch/hoppscotch/pull/2907/commits/e94fb04215071caf6ab5b1910d6df0a6a63294a1#diff-4fea6a6a16cc3be409d941c1eb74d25a2eea0c006e62c53c02a8550c75a57bebL6 . Should we keep it since its not used anywhere?